README.md

```diff
@@ -1,38 +1,97 @@
+<path opacity="0.1" d="M18.0494 17.4471C18.0494 17.4471 16.524 16.4732 15.068 17.4471C14.6159 17.7541 14.3003 18.2253 14.1877 18.7612C14.0902 19.1909 14.0909 19.6371 14.1899 20.0665C14.2888 20.4958 14.4833 20.8971 14.7588 21.2403C14.7588 21.2403 18.664 25.854 19.0291 25.772C19.0362 25.7704 19.0434 25.7687 19.0506 25.767C19.3137 25.7036 21.9622 21.2577 21.9622 21.2577C22.386 20.5226 22.5597 19.6691 22.4571 18.8262C22.3496 17.9961 21.9523 17.121 20.8228 16.821C20.8228 16.821 19.5054 16.2645 18.0494 17.4471Z" fill="black"/>
+<path opacity="0.1" d="M8.52185 24.7386L8.94938 25.4536L9.80436 26.8835L10.0894 27.8844L9.51939 28.7423L10.2319 30.4582L10.5169 31.4591L9.94692 32.031L11.087 32.8889" fill="black"/>
+<path opacity="0.1" d="M29.4705 18.4473V19.5912L28.758 21.45L28.3304 22.4509L29.043 23.7378L29.3281 24.5958L30.4681 25.0247L30.8956 25.7397V27.4555" fill="black"/>
+<path opacity="0.1" d="M36.7385 19.5912L35.5985 20.3061C35.5985 20.3061 35.0284 20.5921 35.0284 20.7351V21.879" fill="black"/>
+<path opacity="0.1" d="M31.6083 15.5876L31.8932 17.0175L31.6083 18.0184" fill="black"/>
+<path opacity="0.1" d="M32.7482 28.4565L31.8932 29.7433L30.8956 30.6012V31.1732L31.4656 31.8881V32.7461L31.0381 33.604" fill="black"/>
+<path opacity="0.1" d="M24.6253 4.14856H23.2002L22.0601 5.14946L20.2075 6.15035L19.3524 6.86533V7.15133L17.4998 8.15223L16.9298 9.15313" fill="black"/>
+<path opacity="0.1" d="M9.23428 6.57916L7.95171 9.15294L7.52418 10.1539L6.52662 11.4407C6.52662 11.4407 5.81407 12.7276 5.67156 12.8706C5.52905 13.0136 4.53149 13.2996 4.53149 13.2996L5.95658 14.7295" fill="black"/>
+<path opacity="0.1" d="M3.53402 16.4454L3.24897 18.3043L3.53402 19.4482V21.45L2.67896 25.1677L3.53402 26.1686L4.53156 27.4555" fill="black"/>
+<path opacity="0.1" d="M5.67163 17.3035L6.52669 17.7324L7.09671 19.5913L8.23682 19.3053L8.80683 20.3062" fill="black"/>
+<path opacity="0.1" d="M21.3475 35.8918L23.6277 35.6058L24.6252 34.6049H26.0503" fill="black"/>
+<path opacity="0.1" d="M7.52417 32.889L8.37923 33.8899L9.80431 34.3189L10.8019 34.6049L11.0869 36.1777" fill="black"/>
+<path opacity="0.1" d="M24.3402 28.0274L25.3378 27.3125L25.7654 26.1685L26.6204 26.0256L27.0479 24.5957" fill="black"/>
+</svg>
 
-First, run the development server:
+</div>
 
-```bash
-npm run dev
-# or
-yarn dev
-# or
-pnpm dev
-```
+##_Shoppingify Client_
+
+---
+
+###Overview
+![Imgur](https://i.imgur.com/GounMss.png)
+
+Shoppingify allows you to take your shopping list wherever you go. With Shoppingify, you can easily create personalized shopping lists for any occasion, from weekly grocery shopping to birthday gifts. You can also organize your lists by categories and add useful notes, such as product sizes and preferences. Shoppingify can be accessed on both mobile devices and computers.
+
+---
+
+###Technologies
+:computer: The technologies chosen for this project are:
```

Comment:
I think that in order to make it look clean we can just add the title of the section and remove everything else.

`### 💻 Technologies`

instead of

`### 💻 Technologies The technologies chosen for this project are:`